src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard/MovieCard';
+import { MovieType } from '.';
```

Comment:
Where are you saves your types file? 
I'm not shure, but i think you should save it in a separate folder if it's interface

Reply:
Try to do it(in "done" commit), but ran into linter error and deside to put types in existing files(what reason for it's exist)

Reply:
You definitely need separate folder for your types